TodoList/TodoList.Api.Tests/Utils/ExtensionItemController.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web.Http;
+using TodoList.Api.Controllers;
+
+namespace TodoList.Api.Tests.Utils
+{
+    public static class ExtensionItemController
```

Comment:
Here again, `ItemControllerExtension` would be a more fitting name (read on, though).